base/REPL.jl

```diff
@@ -416,7 +416,7 @@ function add_history(hist::REPLHistoryProvider, s)
     push!(hist.history, str)
     hist.history_file === nothing && return
     entry = """
-    # time: $(Libc.strftime("%Y-%m-%d %H:%M:%S %Z", time()))
+    # time: $( string( now() ) )
```

Comment:
`string` shouldn't be necessary. I think it's most important that the style matches the rest of the repository. Little bits here and there of different styles ends up looking messy.